org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -476,3 +476,26 @@ GithubOidcSageBionetworksSynapseR:
   DefaultOrganizationBinding:
     Account: !Ref SynapseProdAccount
     Region: us-east-1
+
+GithubOidcNexflowInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-nextflow-infra
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-nextflow-infra
+    MaxSessionDuration: "21600"
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
+      - "arn:aws:iam::aws:policy/AWSKeyManagementServicePowerUser"
+  TemplatingContext:
+    GitHubOrg: "Sage-Bionetworks-Workflows"
+    Repositories:
+      - name: "nextflow-infra"
+        branches: ["dev", "prod"]
```

Comment:
I'm a little wary of giving both branches access to both accounts, can we split dev and prod into separate stacks so that the dev branch doesn't have any access to the prod account like with `synapse-login-aws-infra`?

Reply:
done